secondExample.ts

```diff
@@ -0,0 +1,8 @@
+interface Person {name: string}
+interface Person {age: number}
+
+//We can expand interface to add new fields
+//We can't do this with types in this way
```

Comment:
Types need to be unique in typescript, hence the error (`duplicate identifier`).
Interfaces can be extended in this way (however, in my opinion it's still better to use `extend`, since it makes the code more clear/readable).